examples/pylab_examples/matshow.py

```diff
@@ -10,17 +10,7 @@ def samplemat(dims):
         aa[i, i] = i
```

Comment:
I think the current version is much more readable, which is important for an example.